ras/ras_extended.py

```diff
@@ -249,3 +249,22 @@ def test3_lscfg(self):
         if self.is_fail >= 1:
             self.fail("%s command(s) failed in lscfg tool verification"
                       % self.is_fail)
+
+    def test_usysident(self):
+        """
+        This tests to turn on device identify indicators and other help
+        options of usysident
+        """
+        self.log.info("============Executing usysident tool test============")
+        value = self.params.get('usysident_list', default=['-h', '-V', '-P'])
+        for list_item in value:
+            if 'not supported' in self.run_cmd_out('usysident  %s ' % list_item):
+                self.cancel(
+                    "The identify indicators are not supported on this system")
```

Comment:
what if -P is supported options, if -V is not supported, the whole test gets cancled before testing -P ?